schema.json

```diff
@@ -56,6 +56,7 @@
 				"Hash", "Hash*",
 				"Iterator", "Iterator*",
 				"IteratorSet", "IteratorSet*",
+				"Journal", "Journal*",
```

Comment:
For `Journal` you can use `JournalHandle` because it can be used for a class name. Too common to be using Journal